src/ports/postgres/modules/graph/bfs.py_in

```diff
@@ -125,35 +127,39 @@ def graph_bfs(schema_madlib, vertex_table, vertex_id, edge_table,
                                             default_args)
 
         # Prepare the input for recording in the summary table
-        if vertex_id is None:
-            v_st= "NULL"
+        if (vertex_id is None) or (vertex_id == ''):
+            v_st = "NULL"
```

Comment:
I suggest using `''` to indicate no input. `"NULL"` as a string is different from `NULL`.